### PR TITLE
sysdig 0.32.1

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

- [x] `resource` blocks have been checked for updates.
<details>
  <summary>release notes</summary>
  <pre>## What's Changed
* fix: avoid using global filtercheck list with plugins by @jasondellaluce in https://github.com/draios/sysdig/pull/2008

**Full Changelog**: https://github.com/draios/sysdig/compare/0.32.0...0.32.1</pre>
</details>

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  /usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libelf.a(elf_compress.o): undefined reference to symbol 'ZSTD_decompress'
  /usr/bin/ld: /home/linuxbrew/.linuxbrew/opt/zstd/lib/libzstd.so.1: error adding symbols: DSO missing from command line
```

**Comment by chenrui333** [comment]
seeing the same in #139243

**Comment by alebcay** [comment]
From https://stackoverflow.com/a/55086637,

> The DSO missing from command line message will be displayed when the linker does not find the required symbol with it's normal search but the symbol is available in one of the dependencies of a directly specified dynamic library.

**Comment by chenrui333** [comment]
> From https://stackoverflow.com/a/55086637,
> 
> > The DSO missing from command line message will be displayed when the linker does not find the required symbol with it's normal search but the symbol is available in one of the dependencies of a directly specified dynamic library.

The suggestion works for me. 👍 Thanks for the pointer.

File: Formula/s/sysdig.rb [hunk 1]
```diff
@@ -73,6 +74,10 @@ class Sysdig < Formula
   def install
     (buildpath/"falcosecurity-libs").install resource("falcosecurity-libs")
 
+    # fix `libzstd.so.1: error adding symbols: DSO missing from command line` error
+    # https://stackoverflow.com/a/55086637
+    ENV.append "LDFLAGS", "-Wl,--copy-dt-needed-entries" if OS.linux?
+
```
Comment (ZhongRuoyu) [inline Formula/s/sysdig.rb:80]: Can move this along to make its dependencies ✅. But would be nice to figure out how this happened and how to properly fix it.

**Review by ZhongRuoyu — APPROVED** [review]

**Pull request merged**